bugwarrior/services/jitamin.py

```diff
@@ -0,0 +1,240 @@
+import requests
+import logging
+import typing_extensions
+
+from bs4 import BeautifulSoup
+
+from bugwarrior import config
+from bugwarrior.services import IssueService, Issue, ServiceClient
+
+log = logging.getLogger(__name__)
+
+class JitaminConfig(config.ServiceConfig, prefix="jitamin"):
+    service: typing_extensions.Literal["jitamin"]
+    base_uri: config.StrippedTrailingSlashUrl
+    username: str
+    password: str
+
+    verify_ssl: bool = True
+    column_as_tag: bool = False
+
+    query: str = ""
+
+
+class JitaminIssue(Issue):
+    URL = "jitaminurl"
+    POSITION = "jitaminposition"
+    COLUMN = "jitamincolumn"
```

Comment:
It seems a little odd to have column as a separate UDA and also optionally use it as a tag. Maybe it makes sense to Jitamin users though.

Reply:
No, it was more of a personal preference. I like to see the column in my default task view, which does not include the UDA, so tag seemed to be the first logical value.